include/npcomp/Dialect/Torch/IR/TorchTypes.td

```diff
@@ -164,9 +164,11 @@ def AnyTorchType : AnyTypeOf<[
     AnyTorchScalarType,
     AnyTorchTensorType,
     Basicpy_ListType,
+    Basicpy_TupleType,
     Basicpy_NoneType,
     Basicpy_BytesType,
     Torch_NnModuleType,
+    Torch_OptionalType,
```

Comment:
@silvasean: One thing that having a specific Conv2d test did catch was supporting the optional tensor type as an input argument for a module (since the [bias operand of a conv is optional](https://pytorch.org/docs/stable/generated/torch.nn.Conv2d.html#conv2d)). It's probably safe to assume that our model test(s) will give us this coverage,

Reply:
We could test it with a simple call to torch.nn.function.conv1d in node importer tests instead of a full ivalue importer test. It's not a huge deal. This part of the patch is "obviously correct" IMO.